src/tree/localProject/LocalProjectTreeItemBase.ts

```diff
@@ -7,23 +7,26 @@ import * as path from 'path';
 import { AzExtParentTreeItem, TreeItemIconPath } from 'vscode-azureextensionui';
 import { localize } from '../../localize';
 import { treeUtils } from '../../utils/treeUtils';
+import { isMultiRootWorkspace } from '../../utils/workspace';
 
 export abstract class LocalProjectTreeItemBase extends AzExtParentTreeItem {
     public readonly label: string = localize('localProject', 'Local Project');
     public suppressMaskLabel: boolean = true;
     private readonly _projectName: string;
```

Comment:
nit: Rename to `_projectSubpath` or something similar now that it's not just the name